.travis.yml

```diff
@@ -53,6 +53,15 @@ jobs:
       rust: stable
       env: TASK=release
 
+    # INTERMITTENTLY ALLOWED FAILURES
+    # Allowed if a failure occurs after a new Rust release until the
+    # failure is addressed.
+    - name: "Run clippy on beta toolchain"
+      rust: beta
+      before_script:
+        - rustup compnent add clippy
```

Comment:
There's a typo here.

Reply:
Whoops; I'll fix that with a force-push.